app/controllers/mixins/ems_common_angular.rb

```diff
@@ -831,6 +831,7 @@ def zone
     end
 
     def default_auth_status
+      return true if @ems.kind_of?(ManageIQ::Providers::Kubevirt::InfraManager)
```

Comment:
This is wrong. Please make sure `@ems.authentication_status_ok?` always returns true for `ManageIQ::Providers::Kubevirt::InfraManager`, if you want the button to be enabled unconditionally.

(The Google cloud condition is also wrong, but that's probably out of your hands, right :).)

Reply:
(The reason these are wrong is because the UI should not have to know the details of your provider. That's what the provider models are for.)

Reply:
@himdel Yes, that would be great.

@masayag Would it be possible to implement something like this - https://github.com/ManageIQ/manageiq-providers-ovirt/pull/256 for this case as well in order to address this provider-specfic case?

We are trying our best to reduce provider-specific conditionals in the UI code as far as possible.

Reply:
The google one, I believe was added much earlier even before <a href="https://github.com/ManageIQ/manageiq-providers-google">manageiq-providers-google</a> was born.